Plugin.php

```diff
@@ -236,7 +265,8 @@ public function registerMarkupTags(): array
     {
         return [
             'filters' => [
-                'staticPage' => ['Winter\Pages\Classes\Page', 'url']
+                'staticPage'    => ['Winter\Pages\Classes\Page', 'url'],
+                'parseSnippets' => ['Winter\Pages\Classes\Snippet', 'parse'],
```

Comment:
We need documentation for this filter

Reply:
@LukeTowers I think we should use a modified version of the original `readme.md` file
ref. https://github.com/inetis-ch/oc-richeditorsnippets-plugin

Do you have a suggestion on where to integrate this content ?